config/environments/development.rb

```diff
@@ -81,6 +81,8 @@
 
   # Send mails to browser window
   config.action_mailer.delivery_method = :letter_opener
+
+  config.hosts << 'bs-local.com' if Rails.env.development? && ENV['OPENPROJECT_DISABLE_DEV_ASSET_PROXY'].present?
```

Comment:
You can remove the `Rails.env.development?` call as this file is only evaluated in development mode anyway

Reply:
Ah okay. I could have thought of that myself given the name of the file. 🙈